Documentation/ReleaseNotes.md

```diff
@@ -36,64 +26,6 @@ If importing the [Mixed Reality Toolkit NuGet packages](MRTKNuGetPackage.md), th
 
 - [NuGet for Unity 2.0.0 or newer](https://github.com/GlitchEnzo/NuGetForUnity/releases/latest)
 
-### Upgrading projects to 2.4.0
-
-The 2.4.0 release has some changes that may impact application projects. Breaking change details, including mitigation guidance, can be found in the [**Updating 2.3.0 to 2.4.0**](Updating.md#updating-230-to-240) article.
```

Comment:
Should we still keep a link to `Updating.md` somewhere in here?

Reply:
will add one.

Reply:
done